projects/extensions/datetimepicker/time.html

```diff
@@ -0,0 +1,65 @@
+<div class="mtx-time-row-wrapper">
+  <div class="mtx-time-row">
+    <input 
+      class="mtx-time-input" 
+      [class.mtx-time-input-active]="clockView === 'hour'" 
+      [class.mtx-time-input-warning]="!hourInput.valid"
+      #hourInput="mtxTimeInput" 
+      type="text" 
+      inputmode="numeric" 
+      maxlength="2"
+      (mtxValueChanged)="handleHourInputChange($event)"
+      [mtxValue]="hour"
+      [mtxMin]="twelvehour ? 1 : 0"
+      [mtxMax]="twelvehour ? 12 : 23"
```

Comment:
Maybe the prefix `time` is better than `mtx`.
```
mtxValue => timeValue
mtxValueChanged => timeValueChange
mtxMin => timeMin
mtxMax => timeMax
mtxInterval => timeInterval
```

Reply:
I renamed them